src/Microsoft.TestPlatform.ObjectModel/Constants.cs

```diff
@@ -162,6 +162,11 @@ public static class Constants
         /// </summary>
         public const string ResultsDirectoryName = "TestResults";
 
+        /// <summary>
+        /// Environment variable name to increase communication timeouts between processes.
+        /// </summary>
+        public const string VstestTimeoutIncreaseByTimes = "VSTEST_TIMEOUT_INCREASE_BY_TIMES";
```

Comment:
>VSTEST_TIMEOUT_INCREASE_BY_TIMES [](start = 60, length = 32)

How about VSTEST_CONNECTION_TIMEOUT?
We should have a simple settings, also make it consistent across

Reply:
Makes sense, Done.